buildkite/bazel-central-registry/bcr_presubmit.py

```diff
@@ -473,9 +489,11 @@ def main(argv=None):
         pipeline_steps = []
         for module_name, module_version in modules:
             configs = get_task_config(module_name, module_version)
-            add_presubmit_jobs(module_name, module_version, configs.get("tasks", {}), pipeline_steps)
+            task_configs = duplicate_configs_for_supported_bazel_lts_releases(configs.get("tasks", {}))
+            add_presubmit_jobs(module_name, module_version, task_configs, pipeline_steps)
             configs = get_test_module_task_config(module_name, module_version)
-            add_presubmit_jobs(module_name, module_version, configs.get("tasks", {}), pipeline_steps, is_test_module=True)
+            task_configs = duplicate_configs_for_supported_bazel_lts_releases(configs.get("tasks", {}))
```

Comment:
Not needed - duplicate_...() doesn't mutate the input, so you can just use the original task_configs. If you want to make sure that there are no future side-effects, you could make a copy instead.